doc/ref_model/chapters/chapter01.md

```diff
@@ -4,202 +4,258 @@
+
+    - **Create an additional Architecture only when incompatible elements are unavoidable:**
+      - Creating additional Architectures is limited to when incompatible elements are desired by Taskforce members. For example, if one member desires KVM be used as the hypervisor, and another desires ESXi be used as the hypervisor, and no compromise or mitigation* can be negotiated, the Architecture could be forked, subject to review and vote to approve by the CNTT technical Working Group, such that one Architecture would be KVM-based and the other would be ESXi-based.
+
+        >*Depending on the relationships and substitutability of the component(s) in question, it may be possible to mitigate component incompatibility by creating annexes to a single Architecture, rather than creating an additional Architecture. With this approach, the infrastructure architecture designers might implement the Architecture as described in the reference document, however when there is a potential for incompatibility for  particular component, they would select their preferred option from one of the relevant annexes. For example, if one member wanted to use Ceph, and another member wanted to use Swift, assuming the components are equally compatible with the rest of the Architecture, there could be one annex for the Ceph implementation and one annex for the Swift implementation.
 
 1. NFVI provides abstract and physical resources corresponding to:
    - Compute resources.
    - Storage resources.
-   - Networking resources. (Limited to connectivity services).
+   - Memory resources.
+   - Networking resources. (Limited to connectivity services only).
    - Acceleration resources.
 1. NFVI exposed resources should be supplier independent.
-1. All NFVI APIs must be standard and open to ensure components substitution.
+1. All NFVI Application Programming Interfaces (API) must be standard and open to ensure component substitution if at all possible.
    - NFVI resources are consumed by VNFs through standard and open APIs.
    - NFVI resources are configured on behalf of VNFs through standard and open APIs.
    - NFVI resources are discovered/monitored by management entities (such as orchestration) through standard and open APIs.
-1. VNFs should be modular and utilise minimum resources.
+   - All NFVI APIs must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and come from Open Communities and Standards Development Organizations. Through such APIs will NFVI resources be discovered/monitored by management entities, configured on behalf of VNFs and consumed by VNFs.
+1. VNFs should be modular and be designed to utilise the minimum resources required for the service.
 1. NFVI shall support pre-defined and parameterized T-Shirt sizes.
    - T-Shirt sizes will evolve with time.
```

Comment:
The concept of T-Shirt sizes may not have been introduced yet.

Reply:
Fixed by removing T-shirt reference.